npm/npm_import.bzl

```diff
@@ -295,16 +329,43 @@ def npm_translate_lock(
             wild-cards are not yet supported and npm_translate_lock will fail if there are multiple versions of a package that
             are to be hoisted.
 
-        dev: If true, only install devDependencies
+            ```
+            public_hoist_packages = {
+                "@foo/bar": [""]
+                "fum@0.0.1": ["some/sub/package"]
+            },
+            ```
+
+            List of public hoist packages are additive when there are multiple matches for a package.
+
+        dev: If True, only install devDependencies
+
+        no_optional: If True, optionalDependencies are not installed
```

Comment:
opt: link to that issue about platform-specific optional deps?

Reply:
✅